server/modules/category/repository/categoryRepository.js

```diff
@@ -69,7 +69,28 @@ export const insertItems = async (surveyId, items, client = db) => {
 
 // ============== READ
 
-const _getFetchCategoriesAndLevelsQuery = ({ surveyId, draft, includeValidation, offset = null, limit = null }) => `
+const _getFetchCategoriesAndLevelsQuery = ({
+  surveyId,
+  draft,
+  includeValidation,
+  backup = false,
+  offset = null,
+  limit = null,
+}) => {
+  const propsFields = (tableAlias) => {
+    if (backup) {
+      // keep both props and propsDraft
+      const fields = [`'props', ${tableAlias}.props`]
+      if (draft) {
+        fields.push(`'propsDraft', ${tableAlias}.props_draft`)
+      }
```

Comment:
if backup, don't we always include propsDraft?

Reply:
right